pilot/pkg/model/virtualservice.go

```diff
@@ -581,57 +581,3 @@ func UseIngressSemantics(cfg config.Config) bool {
 func UseGatewaySemantics(cfg config.Config) bool {
 	return cfg.Annotations[constants.InternalRouteSemantics] == constants.RouteSemanticsGateway
 }
-
-// VirtualServiceDependencies returns dependent configs of the vs,
-// for internal vs generated from gateway-api routes, it returns the parent routes,
-// otherwise it just returns the vs as is.
-func VirtualServiceDependencies(vs config.Config) []ConfigKey {
```

Comment:
No longer needed. Was added in https://github.com/istio/istio/pull/40692, now we send an event for the VS itself